migrations/scripts/README.md

```diff
@@ -65,7 +65,7 @@ Or
 
 ```sh
 # helm 3
-helm3 list <release name> -n <namespace>
+helm list <release name> -n <namespace>
```

Comment:
Do we still need the helm2 section above this?

Reply:
Good catch, it should be deleted.

Reply:
My bad, there was a space in between "helm" and "2/3" so search I ran missed it! :( Thank you both for picking it up.